docs/https.md

```diff
@@ -0,0 +1,32 @@
+# HTTPS
+By default MailDev will run on the http protocol. For Web Notification support you'll need
```

Comment:
One can also use web notification support on localhost in most browsers. We should update this copy.

Reply:
Ah. I wasn't aware localhost was the exception. Indeed useful to be added.